walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -58,43 +59,40 @@
         }
       }
 
-      /** If the event starts in the beginin of the day, make the 
-        * first possible start at the end of this event. 
-        */
+      // If the event starts in the beggining of the day, make the 
+      // first possible start at the end of this event. 
       if (time.start() == 0) {
         start = time.end();
       }
       end = time.start();
-      freeTime = end - start;
 
-      if (freeTime > 0 && request.getDuration() <= freeTime) {
+      if (freeTime(start, end) > 0 && request.getDuration() <= freeTime(start, end)) {
         meetingTimes.add(TimeRange.fromStartEnd(start, end, false));
       }
-
       start = time.end();
     }
 
     end = TimeRange.END_OF_DAY;
 
-    if (request.getDuration()<= end - start) {
+    if (request.getDuration() <= freeTime()) {
       meetingTimes.add(TimeRange.fromStartEnd(start, end, true));
     }
     return new ArrayList<TimeRange>(meetingTimes);
-    // return meetingTimes;
-    // throw new UnsupportedOperationException("TODO: Implement this method.");
   }
 
-  /** Return true iff any person in {@code inEvents} are contained in
-    * {@code required}. *
+  /** 
+    * Return true iff any person in {@code inEvents} are contained in
+    * {@code required}.
     */
```

Comment:
```suggestion
  /** 
   * Return true iff any person in {@code inEvents} are contained in
   * {@code required}.
   */
```

Also add docs for the params